Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1

```diff
@@ -77,7 +77,7 @@ function Get-TeamByName
     $loopCounter = 0
     do
     {
-        $team = Get-Team -DisplayName $TeamName
+        $team = Get-Team -DisplayName $TeamName | Where-Object -FilterScript {$_.DisplayName -eq $TeamName}
```

Comment:
Parameters look the same, both checking DisplayName against $TeamName. Shouldn't it be either the parameter or the Where-Object?

Reply:
Get-Team -DisplayName, based on documentation is a wild card search. In our case we want an exact match:
https://docs.microsoft.com/en-us/powershell/module/teams/get-team?view=teams-ps#example-4